model_inference/bird_predictor.py

```diff
@@ -17,12 +17,18 @@
         6: 'Цыпленок'
     }
 
+def download_file_from_url(url, destination_path):
```

Comment:
Let's try adding typing to all functions.
How, for example, typing might look like in this case:
def download_file_from_url(url: str, destination_path: str) -> None: